### DISCUSSION TITLE
V1.35.1 history data cannot be shown

### DISCUSSION THREAD

**Discussion by cfangplus** [netdata/netdata#15885] [Q&A] [answered]
hi community

Now I installed netdata with 1.35.1 version, after installation I come to the localhost:19999 page and check to see the working status of CPU/Memory/Network but I faild to see the streaming stats and only shows a small part of them, that's to say the history data can't be showed. Also I found there are words appeared in each metric says "Want to extend your history of real-time metrics? Configure NetData's history or use the DB engine". It seems that I have not configured the netdata.conf file, so I jump to /opt/netdata/etc/netdata/ and configure netdata.conf as follow:
[global]  
            run as user = netdata
            page cache size = 32
            dbengine multihost disk space = 256
However, the streaming stats still can't be showed and I cann't see the history status. Need you help, thx.

**Comment by MrZammler** [comment]
Hi @cfangplus 

Can you check `http://localhost:19999/api/v1/info` ? There should be an item `memory-mode` which shows if you're using dbengine or not.

I'm a bit confused when you say `streaming` stats, what exactly do you mean? For Netdata, streaming refers to when an agent (a child) streams it's metrics to another agent (the parent).

**Reply by cfangplus** [reply]
about so called 'streaming', I mean the history data, like data flow.
I checked memory-mode item from http://localhost:19999/api/v1/info and it is dbengine, that means dbengine is configured but I still cannot see history metric data.

**Comment by cfangplus** [comment]
when the first time opened localhost:19999 page, there is a hint called 'we couldnot load the latest netdata ui', after click the 'old single node dashboard' i found this problem that's mentioned before.
![IMG_20230828_220824](https://github.com/netdata/netdata/assets/42287875/90d0da91-99f9-44b1-853a-bdcd6f86b46f)

**Comment by MrZammler** [comment, marked as answer]
Hi @cfangplus 

In general, 1.35.1 is a bit old version.... is there a way to update to 1.42.1 ?

**Comment by cfangplus** [comment]
fine，it's ok
